pkg/aws/ec2/ec2.go

```diff
@@ -617,6 +619,55 @@ func (c *Client) GetSubnets(ctx context.Context) (ipamTypes.SubnetMap, error) {
 	return subnets, nil
 }
 
+// describeRouteTables lists all route tables
+func (c *Client) describeRouteTables(ctx context.Context) ([]ec2_types.RouteTable, error) {
+	var result []ec2_types.RouteTable
+	input := &ec2.DescribeRouteTablesInput{}
+	if len(c.routeTableFilters) > 0 {
+		input.Filters = c.routeTableFilters
+	}
+	paginator := ec2.NewDescribeRouteTablesPaginator(c.ec2Client, input)
+	for paginator.HasMorePages() {
+		c.limiter.Limit(ctx, DescribeRouteTables)
+		sinceStart := spanstat.Start()
+		output, err := paginator.NextPage(ctx)
+		c.metricsAPI.ObserveAPICall(DescribeRouteTables, deriveStatus(err), sinceStart.Seconds())
+		if err != nil {
+			return nil, err
+		}
+		result = append(result, output.RouteTables...)
+	}
+	return result, nil
+}
+
+// GetRouteTables returns all EC2 route tables as a RouteTableMap
+func (c *Client) GetRouteTables(ctx context.Context) (ipamTypes.RouteTableMap, error) {
+	routeTables := ipamTypes.RouteTableMap{}
+
+	routeTableList, err := c.describeRouteTables(ctx)
+	if err != nil {
+		return nil, err
```

Comment:
I would have probably wrap the error here, just adding that we are trying to get the route tables.

Reply:
I'm not sure since GetSubnets and GetVpcs are throwing this out? Shall I change all of them?